lib/importers/schema_dot_org_json_data.rb

```diff
@@ -67,4 +68,9 @@ def process_organization_name(job)
     return nil
   end
 
+  def has_minimum_required_props(job_posting)
+    required_properties = ['hiringOrganization','datePosted', 'url']
+    required_properties.map {|x| !job_posting[x].blank?}.all?
```

Comment:
this is odd.  i thought i changed it to "prop."  better check that all my changes made it into this push.